### PR TITLE
changes made

### PR DESCRIPTION
**Pull request by dbaskak**

### CODE REVIEW

File: lesson_04/homework_04.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+""" Виведіть, скількі разів у тексті зустрічається літера "h"
+"""
+number_of_h = adwentures_of_tom_sawer.count("h")
+print(number_of_h)
+
+# task 05
+""" Виведіть, скільки слів у тексті починається з Великої літери?
+"""
+upper_case_letters = sum(1 for letter in adwentures_of_tom_sawer if letter.isupper())
+print(upper_case_letters)
```
Comment (edanilovets) [inline lesson_04/homework_04.py:49]: The task is about to check what number of WORDS start with the uppercase letter. This solution is working for current task (but it is not checking only words first letter), but will not work for words like "iPhone, eBay"...

File: lesson_04/homework_04.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Перетворіть рядок у нижній регістр.
+"""
+print(adwentures_of_tom_sawer_sentences[3].lower())
+
+# task 09
+""" Перевірте чи починається якесь речення з "By the time".
+"""
+start_with = "By the time"
+position = adwentures_of_tom_sawer.find(start_with)
+print(f"Sentence 'By the time' starts at {position} position")
```
Comment (edanilovets) [inline lesson_04/homework_04.py:76]: The task 09 is asking to output True or False, not the index

File: lesson_04/homework_04.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+# task 10
+""" Виведіть кількість слів останнього речення з adwentures_of_tom_sawer_sentences.
+"""
+count = len(adwentures_of_tom_sawer_sentences[-1].split())
+# print(adwentures_of_tom_sawer_sentences[1])
```
Comment (edanilovets) [inline lesson_04/homework_04.py:82]: last sentence should contains letters.
 and len should not be 0.
 Try another split() param in task 07

**Comment by dbaskak** [comment]
пт, 30 серп. 2024 р. о 14:43 Eugene Danilovets ***@***.***>
пише:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738476331>:
>
> > +upper_case_letters = sum(1 for letter in adwentures_of_tom_sawer if letter.isupper())
> +print(upper_case_letters)
>
> The task is about to check what number of WORDS start with the uppercase
> letter. This solution is working for current task (but it is not checking
> only words first letter), but will not work for words like "iPhone, eBay"...
>
Corrected.

> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738477695>:
>
> > +# task 09
> +""" Перевірте чи починається якесь речення з "By the time".
> +"""
> +start_with = "By the time"
> +position = adwentures_of_tom_sawer.find(start_with)
> +print(f"Sentence 'By the time' starts at {position} position")
>
> The task 09 is asking to output True or False, not the index
>
Corrected.

> ------------------------------
>
> In lesson_04/homework_04.py
> <https://github.com/dbaskak/aqa_01/pull/3#discussion_r1738479104>:
>
> > +# task 10
> +""" Виведіть кількість слів останнього речення з adwentures_of_tom_sawer_sentences.
> +"""
> +count = len(adwentures_of_tom_sawer_sentences[-1].split())
> +# print(adwentures_of_tom_sawer_sentences[1])
>
> last sentence should contains letters.
> and len should not be 0.
> Try another split() param in task 07
>
Corrected
Thanks for you time

**Comment by edanilovets** [comment]
@dbaskak When you updated PR , please click Resolve Conversation buttons. And it is a good practice to answer directly on each conversation
Screenshot 2024-08-31 at 15 57 50

**Comment by dbaskak** [comment]
Ok, will keep it in mind for further

File: lesson_04/homework_04.py (hunk 3 above)
Comment (dbaskak) [reply lesson_04/homework_04.py:82]: ok

File: lesson_04/homework_04.py (hunk 1 above)
Comment (dbaskak) [reply lesson_04/homework_04.py:49]: ok

**Pull request merged**